src/main/resources/application.yml

```diff
@@ -26,9 +26,12 @@ spring:
             token-uri: ${api.base.url.oauth}/oauth/token
 
   jackson:
-    date-format: "yyyy-MM-dd HH:mm:ss"
```

Comment:
this is not how we want dates formatted. in fact, this is not even how they _were_ formatted because this setting only applies to `java.util.Date`.